<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QComboBox" name="comboBox_project">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>201</width>
      <height>28</height>
     </rect>
    </property>
   </widget>
   <widget class="QTabWidget" name="tabWidget_asset">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>80</y>
      <width>751</width>
      <height>411</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Asset</string>
     </attribute>
     <widget class="QListWidget" name="listWidget_category">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>151</width>
        <height>361</height>
       </rect>
      </property>
     </widget>
     <widget class="QTableWidget" name="tableWidget_asset">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>10</y>
        <width>211</width>
        <height>361</height>
       </rect>
      </property>
      <attribute name="horizontalHeaderVisible">
       <bool>false</bool>
      </attribute>
      <attribute name="horizontalHeaderDefaultSectionSize">
       <number>190</number>
      </attribute>
      <attribute name="verticalHeaderVisible">
       <bool>false</bool>
      </attribute>
      <attribute name="verticalHeaderDefaultSectionSize">
       <number>85</number>
      </attribute>
      <attribute name="verticalHeaderHighlightSections">
       <bool>true</bool>
      </attribute>
     </widget>
     <widget class="QListWidget" name="listWidget_tasks">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>10</y>
        <width>121</width>
        <height>181</height>
       </rect>
      </property>
     </widget>
     <widget class="QTableWidget" name="tableWidget_scenes">
      <property name="geometry">
       <rect>
        <x>490</x>
        <y>10</y>
        <width>251</width>
        <height>361</height>
       </rect>
      </property>
      <attribute name="horizontalHeaderVisible">
       <bool>false</bool>
      </attribute>
      <attribute name="horizontalHeaderDefaultSectionSize">
       <number>250</number>
      </attribute>
      <attribute name="verticalHeaderVisible">
       <bool>false</bool>
      </attribute>
      <attribute name="verticalHeaderDefaultSectionSize">
       <number>100</number>
      </attribute>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Seq</string>
     </attribute>
     <widget class="QListWidget" name="listWidget_sequence">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>201</width>
        <height>381</height>
       </rect>
      </property>
     </widget>
     <widget class="QTableWidget" name="tableWidget_shot">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>0</y>
        <width>221</width>
        <height>381</height>
       </rect>
      </property>
     </widget>
     <widget class="QTableWidget" name="tableWidget_thumbnail">
      <property name="geometry">
       <rect>
        <x>570</x>
        <y>0</y>
        <width>191</width>
        <height>381</height>
       </rect>
      </property>
     </widget>
     <widget class="QListWidget" name="listWidget_task">
      <property name="geometry">
       <rect>
        <x>420</x>
        <y>0</y>
        <width>151</width>
        <height>381</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
